src/webui/www/private/scripts/prop-trackers.js

```diff
@@ -253,6 +271,28 @@ window.qBittorrent.PropTrackers ??= (() => {
             });
     };
 
+    const reannounceTrackersFN = (element, allTrackers = false) => {
```

Comment:
Maybe make the second argument non-optional? So the action would be clearer to the code reader.
```suggestion
    const reannounceTrackersFN = (element, allTrackers) => {
```